src/main/java/de/tonypsilon/bmm/backend/match/data/Match.java

```diff
@@ -43,17 +43,17 @@ public class Match {
      * Note: To avoid strange round off behaviour for half points, points are stored
      * at the doubled amount to guarantee that they are integers.
      */
-    @Column(name = "overruled_home_board_half_points", unique = false, nullable = true)
```

Comment:
nullable = true can be removed here as well.
In IntelliJ, these issues can be found be going right click on the source foulder, -> "analyze" -> "inspect code". In the results tab, the exact locations are shown at Java > Declaration redundancy > Default annotaion parameter value.
![image](https://user-images.githubusercontent.com/70838192/233093901-a96af1cb-8ddd-479b-9336-78195a883205.png)
(Here is how it looks like for me)

Reply:
Thanks for this @Tonypsilon , didn't know about this until now

I am using IntelliJ but in 2 classes having Default annotation parameter value, have removed those and commited
![image](https://user-images.githubusercontent.com/106817047/233117889-bb11b8ee-e306-406a-8d83-45e6419dc594.png)

Reply:
Thanks for this @Tonypsilon , didn't know about this until now

Yes I am using IntelliJ , have used "analyze" and removed Default annotation parameter values and committed PR#12

Reply:
Thanks for this @Tonypsilon , didn't know about this until now

Yes I am using IntelliJ , have used "analyze" and removed Default annotation parameter values and committed PR #12

Reply:
Thanks for this @Tonypsilon , didn't know about this until now

Yes I am using IntelliJ , have used "analyze" and removed Default annotation parameter values and committed, have a look!